### PR TITLE
Created paint-order example in HTML

### LINKED ISSUE #32236: paint-order also applies to HTML content (not only SVG)
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/paint-order

### What specific section or headline is this issue about?

Examples

### What information was incorrect, unhelpful, or incomplete?

`paint-order` is already supported for HTML content in Firefox and Safari, and the feature [is coming to Chrome](https://groups.google.com/a/chromium.org/g/blink-dev/c/-eflVqNCtDY/m/jPnpYMhMAAAJ). It might be nice to include a example without SVG in this page.

### What did you expect to see?

A new example without using SVG for this property.
Maybe we need even a new entry in the browser compatibility table, as support in non SVG content has come later to browsers.

### Do you have any supporting links, references, or citations?

Intent-to-ship: https://groups.google.com/a/chromium.org/g/blink-dev/c/-eflVqNCtDY/m/jPnpYMhMAAAJ
Example (works in Firefox and Safari or with the runtime flag in Chrome): https://jsfiddle.net/4mh71efb/

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/paint-order`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/paint-order
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/paint-order/index.md
* Last commit: https://github.com/mdn/content/commit/0b26e0b94c77c3d34f0b8b77ba624a7323ccfdad
* Document last modified: 2023-04-07T05:18:00.000Z

</details>

### PR DESCRIPTION
**Pull request by srikanth-kandi**
### Description

Changed the SVG example for `paint-order` to HTML example using `<div>` tags and modified the CSS attributes `stroke` and `stroke-width` to `-webkit-text-stroke-color` and `-webkit-text-stroke-width`.

### Related issues and pull requests

Fixes #32236

https://github.com/mdn/content/assets/87417638/1295abe5-b119-4c7e-a25c-5f16940c8830

### CODE REVIEW

**Review by bsmth — CHANGES_REQUESTED** [review]
Thanks a lot for opening this one @srikanth-kandi 

It would be great if we cover both cases for HTML & SVG instead of replacing SVG entirely. Do you want to add your example as an addition so we have two - one for HTML and one for SVG?

**Comment by srikanth-kandi** [comment]
> It would be great if we cover both cases for HTML & SVG instead of replacing SVG entirely. Do you want to add your example as an addition so we have two - one for HTML and one for SVG?

But what might be the order of the examples, SVG first HTML second or vice versa?

**Comment by srikanth-kandi** [comment]
> It would be great if we cover both cases for HTML & SVG instead of replacing SVG entirely. Do you want to add your example as an addition so we have two - one for HTML and one for SVG?

For now, I placed SVG first and HTML example second here is the expected output below

https://github.com/mdn/content/assets/87417638/db1159b0-89ec-4956-85a2-06dd651cc63e

**Comment by bsmth** [comment]
> For now, I placed SVG first and HTML example second here is the expected output below

This looks great to me, thanks.

File: files/en-us/web/css/paint-order/index.md [hunk 1]
```diff
@@ -88,6 +88,38 @@ text {
 
 {{EmbedLiveSample("Reversing_the_paint_order_of_stroke_and_fill", "100%", 165)}}
 
+### Reversing the paint order of stroke and fill using HTML
+
```
Comment (bsmth) [inline files/en-us/web/css/paint-order/index.md:92]:
```suggestion

To control the fill and stroke order in HTML, you can use the {{cssxref("-webkit-text-stroke-color")}} and {{cssxref("-webkit-text-stroke-width")}} CSS properties.

```

**Review by bsmth — APPROVED** [review]
Thanks a lot. There's one outstanding suggestion for you, but I'm leaving a +1. Well done on your first contribution - welcome aboard 🎉

**Comment by srikanth-kandi** [comment]
hey @bsmth, could you merge this pull request :)

**Comment by bsmth** [comment]
> hey @bsmth, could you merge this pull request :)

Thanks for the reminder, merging now!

**Pull request merged**